cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamFileJanitorService.java

```diff
@@ -68,7 +68,6 @@ public void run() {
           LOG.debug("Completed stream file cleanup.");
         } catch (Throwable e) {
           LOG.warn("Failed to cleanup stream file: {}", e.getMessage());
-          LOG.debug("Failed to cleanup stream file.", e);
```

Comment:
I think the purpose may have been to log the entire exception as debug, and just the message as warn.